src/platforms/react-native/troubleshooting.mdx

```diff
@@ -168,3 +168,64 @@ If you're using `React Navigation 5` and you're not receiving transactions anymo
 ## RAM Bundles Mismatched Line Numbers
 
 If you experience mismatched line numbers on [sentry.io](https://sentry.io) when using RAM Bundles, this is due to a [bug](https://github.com/facebook/metro/issues/399) on the Metro tooling.
+
+## React Native 0.69 and higher
+
+If you're using React Native v0.69 or higher, the auto-linking has changed and we're working on a fix, follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2267).
+
+As a workaround, for now, update Sentry's `react-native.config.js` file located in `node_modules/@sentry/react-native/react-native.config.js`.
+
+_Old_:
+
+```js
+module.exports = {
+  dependency: {
+    platforms: {
+      ios: {
+        sharedLibraries: ['libz']
+      },
+      android: {
+        packageInstance: 'new RNSentryPackage()'
+      }
+    },
+    hooks: {
+      postlink:
+        'node node_modules/@sentry/wizard/dist/bin.js -i reactNative -p ios android',
+      postunlink:
+        'node node_modules/@sentry/wizard/dist/bin.js -i reactNative -p ios android --uninstall'
+    }
+  }
+};
+```
+
+_New_:
+
+```js
+module.exports = {
+  dependency: {
+    platforms: {
+      ios: {},
+      android: {
+        packageInstance: 'new RNSentryPackage()'
+      }
+    }
+  }
+};
+```
+
+This should work on Android, however, on iOS, it'd still fail in the `Bundle React Native code and images` build step phase and for that, you've to revert the patched `shellScript` in the `ios/$projectName.xcodeproj/project.pbxproj` file where `$projectName` is likely your project's name.
+Search for the build step phase `Bundle React Native code and images` and patch the `shellScript`.
+
+_Old_:
+
+```bash
+shellScript = "export SENTRY_PROPERTIES=sentry.properties\nexport EXTRA_PACKAGER_ARGS=\"--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map\"\nset -e\n\nWITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"\n../node_modules/@sentry/cli/bin/sentry-cli react-native xcode REACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"\n\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
```

Comment:
Or:
`shellScript = "set -e\n\nexport NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";`
If RN is lower than 0.69 and not patched by Sentry yet.